checks/evaluation/security_policy.go

```diff
@@ -28,6 +28,7 @@ func SecurityPolicy(name string, dl checker.DetailLogger, r *checker.SecurityPol
 
 	// Apply the policy evaluation.
 	if r.Files == nil || len(r.Files) == 0 {
+		// if file is null or file has zero lengths, directly return not detected
```

Comment:
nit: start your comment with capital letter, and end with a full dot

Reply:
modified, thanks